app/_posts/blog/2015-04-27-state-of-the-moustache-2015.md

```diff
@@ -0,0 +1,120 @@
+
+[How to set up a modern web app and stay sane](http://khmylov.com/blog/2015/02/build-web-app-and-survive/)
+
+[How to Wire Up Ruby on Rails and AngularJS as a Single-Page Application (Gulp Version)](http://angularonrails.com/how-to-wire-up-ruby-on-rails-and-angularjs-as-a-single-page-application-gulp-version/?utm_medium=email&utm_source=rubyweekly)
+
+[Practical TypeScript Development with Gulp and Sublime Text 3](https://airpair.com/typescript/posts/typescript-development-with-gulp-and-sublime-text?utm_medium=email&utm_source=javascriptweekly)
+
+### Some generators we liked
+
+[A new generator for Django projects](http://axiacore.github.io/generator-django-axiacore/)
+
+[Mobile Chrome Apps Starter Kit with Polymer](http://goo.gl/3MdaIk)
+
+[Angular + Gulp generator has a new release](https://github.com/Swiip/generator-gulp-angular/releases)
+
+[Symphony2 with JSPM and critical-CSS optimisation](https://github.com/bezoerb/generator-grunt-symfony)
+
+[Generator for Adobe Brackets extensions](https://github.com/sixertoy/generator-brackextension)
+
+## Announcements from core
+
+### Exploring npm3
+
+[npm@3](https://github.com/npm/npm/wiki/Roadmap) plans to introduce a number of improvements, including a caching rewrite and no longer installing `peerDependencies` by default. This particular change will have an impact on `yo` as it means you end up needing to resolve peerDep conflicts yourself manually. We may end up having our sub-generators in `dependencies` instead, but look forward to taking full advantage of npm@3 where it'll benefit our users.
+
+### Deprecations
+
+As Simon mentioned in our [0.19 release notes](https://github.com/yeoman/generator/releases/tag/v0.19.0) for the generator system, we're currently looking to deprecate a set of methods as we work on cleaning up our API towards an eventual 1.0 release. As always, please test your generators against the latest version of `yeoman-generator` to ensure everything continues to run smoothly.
+
+### Composition FTW
+
+It's been pleasant to see an [increasing](https://twitter.com/tomwayson/status/588564830665084930) number of developers using our [composability](http://yeoman.io/authoring/composability.html) feature for their generators. This enables building upon common ground so that a generator can be run inside another generator, rather than having to replicate functionality. E.g call out to a generator for scaffolding Karma tests rather than doing this inside your AngularApp one.
+
+## Tipping our hat
+
+That's about it for this round-up. We would like to remind folks that we're always checking out [@yeoman](http://twitter.com/yeoman) and enjoy hearing suggestions for anything we can be doing better, news, new generators you've written or even just [doodles](https://twitter.com/kittytail38/status/573522972528373761) of your favorite man in a hat. Until next time, `yo awesomeness`.
+
```

Comment:
Maybe encourage people to comment interesting stuff for the next post here: https://github.com/yeoman/yeoman/issues/1411